validator/sawtooth_validator/gossip/gossip_handlers.py

```diff
@@ -246,10 +261,8 @@ def handle(self, connection_id, message_content):
             if not self._completer.get_block(obj.header_signature):
                 self._gossip.broadcast_block(obj, exclude, time_to_live=ttl)
         elif tag == GossipMessage.CONSENSUS:
-            self._notifier.notify_peer_message(
-                message=obj,
-                sender_id=bytes.fromhex(
-                    self._gossip.peer_to_public_key(connection_id)))
+            # We do not want to forward consensus messages
```

Comment:
Is it true that we don't want to forward consensus messages? We need to be careful here not to apply assumptions about a fully connected network to the general behavior.

Reply:
It's been a simplifying assumption we've made up to this point because all of the engines we've implemented that use consensus messages are supposed to be fully peered (AFAIK). But it does seem like a bad assumption to make in general.

Reply:
POET is not fully peered. But POET doesn't use consensus messages now